dts/bindings/adc/adi,ad4114-adc.yaml

```diff
@@ -0,0 +1,13 @@
+include: [adc-controller.yaml, spi-device.yaml]
+
+compatible: "adi,ad4114-adc"
+
+description: >
```

Comment:
For single line descriptions, it can be kept single line `description: Binding for the ADI AD4114 ADC.`

Reply:
Yes, I have updated the description